preconditions-assertj/src/main/java/com/palantir/logsafe/testing/LoggableExceptionAssert.java

```diff
@@ -24,11 +24,16 @@
 import org.assertj.core.api.ListAssert;
 import org.assertj.core.util.Objects;
 
-public class LoggableExceptionAssert<T extends Throwable & SafeLoggable>
+public final class LoggableExceptionAssert<T extends Throwable & SafeLoggable>
         extends AbstractThrowableAssert<LoggableExceptionAssert<T>, T> {
     private final ArgsAssert argsAssert;
 
-    public LoggableExceptionAssert(T actual) {
+    static <T extends Throwable & SafeLoggable> LoggableExceptionAssert<T> create(T actual) {
+        return new LoggableExceptionAssert<>(actual)
+                .withRepresentation(LoggableArgRepresentation.INSTANCE);
+    }
+
+    private LoggableExceptionAssert(T actual) {
```

Comment:
We can keep this public to avoid breaks, but we probably didn't want it to be public in the first place